bioimageio/spec/v0_3/__init__.py

```diff
@@ -1,12 +1,15 @@
-from bioimageio.spec.shared import fields
+from bioimageio.spec.shared import fields, get_args as _get_args
 from . import nodes, raw_nodes, schema, utils
-from .converters import maybe_convert_manifest, maybe_convert_model
+from .converters import maybe_convert_manifest, maybe_convert_model, maybe_update_model
 from .raw_nodes import ModelFormatVersion
 
 fields = fields
 
 
-get_nn_instance = utils.get_nn_instance
-download_uri_to_local_path = utils.download_uri_to_local_path
-load_raw_model = utils.load_raw_model
-load_model = utils.load_model
+load_raw_model = utils.IO.load_raw_model
+load_model = utils.IO.load_model
+save_raw_model = utils.IO.save_raw_model
+serialize_raw_model_to_dict = utils.IO.serialize_raw_model_to_dict
+export_package = utils.IO.export_package
```

Comment:
current version of `export_package` exposed at `bioimageio.spec.export_package` 
thanks to @k-dominik this is autogenerated from `bioimageio.spec.v0_3.export_package`